README.md

```diff
@@ -3,29 +3,184 @@
 This tool can be used to generate an isosurface geometry from any 3D mesh
 tagged with scalar values (eg, a Cartesian weight window mesh used for
 Monte Carlo particle transport).
-The tool will create a meshed surface geometry using specified isosurface
+The tool will create a meshed surface geometry ([DAGMC](https://svalinn.github.io/DAGMC/)-compliant) using specified isosurface
 values.
 
-## Dependencies:
+## Table of Contents
+
+1. [Installation](#Installation)
+2. [Python Module](#Python-Module-Usage)
+3. [Command Line Tool](#Command-Line-Tool)
+-----
+
+## Installation
+
+### Dependencies
 
 * Python 2.7
 * [VisIt](https://wci.llnl.gov/simulation/computer-codes/visit/)
 * [MOAB](https://sigma.mcs.anl.gov/moab-library/) v5.1+ with PyMOAB enabled
 
-## Installation:
+### Pip install
+
+From the source directory, run the following to install:
+
+      pip install . --user
+
+---
+
+## Python Module Usage
+
+To use, import the Python module with `from IsogeomGenerator import vol`.
+
+### Steps
+
+To generate a full geometry from a starting Cartesian mesh file, run the following steps in order:
+
+1. **Set contour levels values:** This will assign the values that will be used for the isosurface selections
+    in the mesh file. The values can be set using one of three methods:
+
+    * `assign_levels(levels)`: User defines the contour levels to be used as the isosurfaces.
+
+        Input: `levels`, list of floats, list of user-defined values
+
+    * `read_levels(levelfile)`: Read level values from a file. One value per line only.
+
+        Input: `levelfile`, string, relative path to file with level information.
+
+    * `generate_levels(N, minN, maxN, mode='lin')`: Auto-generate evenly-spaced level values between the min and max value.
```

Comment:
isn't that line a bit long ?

Reply:
READMEs don't follow PEP8 line length requirements.. no it is not too long because it auto wraps when rendered.